packages/stress/scripts/localhost_schemachat.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -euo pipefail
+cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
+cd ..
+
+# run scripts/localhost_chat_setup.sh to set up the environment variables
+
+# List of environment files to source
+ENV_FILES=(
+    "./scripts/.env.storage"
```

Comment:
These tests don't use environment variables besides storage. All test-specific config (space id, channel ids) I plan to move into the test schema, or make available via orchestration.